docker/release/dockerfiles/opensearch.al2.dockerfile

```diff
@@ -106,4 +105,4 @@ LABEL org.label-schema.schema-version="1.0" \
   org.label-schema.build-date="$BUILD_DATE"
 
 # CMD to run
-CMD ["./opensearch-docker-entrypoint.sh"]
+ENTRYPOINT ["./opensearch-docker-entrypoint.sh"]
```

Comment:
Please pull from main as we have already made a lot of changes to the entrypoint file at this point.